src/router/index.ts

```diff
@@ -9,15 +10,8 @@ const router = createRouter({
       name: 'tiny-mce',
       component: TinyMCE
     },
-    // {
-    //   path: '/about',
```

Comment:
The changes look good overall. The commented-out route has been removed, which improves code cleanliness. A comment has been added to explain the router creation. However, the comment about additional routes could be more specific.

``` 
// Add new routes here as needed
```